kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -192,19 +191,31 @@ void AddGeometricalUtilitiesToPython(pybind11::module &m)
         .def("CreateNodesArray", &IsosurfacePrinterApplication::CreateNodesArray)
         ;
 
-    //binbased locators
+    // BinBasedFastPointLocator
     py::class_< BinBasedFastPointLocator < 2 >, BinBasedFastPointLocator < 2 >::Pointer >(m,"BinBasedFastPointLocator2D")
         .def(py::init<ModelPart& >())
         .def("UpdateSearchDatabase", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabase)
         .def("UpdateSearchDatabaseAssignedSize", &BinBasedFastPointLocator < 2 > ::UpdateSearchDatabaseAssignedSize)
         .def("FindPointOnMesh", &BinBasedFastPointLocator < 2 > ::FindPointOnMeshSimplified)
```

Comment:
What's the difference with this one?

Reply:
answered below (in short, the previous one is unusable or at least very difficult to use)